flink-runtime/src/test/java/org/apache/flink/runtime/operators/testutils/MockEnvironmentBuilder.java

```diff
@@ -154,6 +159,12 @@ public MockEnvironment build() {
 			subtaskIndex,
 			userCodeClassLoader,
 			taskMetricGroup,
-			taskManagerRuntimeInfo);
+			taskManagerRuntimeInfo,
+			memoryManager);
+	}
+
+	public MockEnvironmentBuilder setMemoryManager(MemoryManager memoryManager) {
```

Comment:
nit: actually this is not used now in all places and we can add it by demands future. If we want to rich the builder now, it is better to place it in front of `#build()` to make related methods close with each other.